### DISCUSSION TITLE
[BUG] Apollo useQuery works with virtualization?

### DISCUSSION THREAD

**Discussion by javierdev-pro18** [petyosi/react-virtuoso#827] [Q&A]
**Describe the bug**
Is Apollo `useQuery` compatible to work with react-virtuoso probably is so slow to work with virtualize lists? when I fetch multiple requests when scroll and wrap the code with Virtuoso, the network cancel most of the requests, making the implementation blinking, is the expected behavior?
When I fetched all the data in only one service and pass to react-virtuoso it works properly.

**Expected behavior**
The cancellation of requests when we fetch the data until scroll make the UI blinking, when we pass all the data already fetched virtuoso works fine, is there any issues with it?

**Screenshots**
<img width="1484" alt="Screen Shot 2023-01-20 at 10 22 08 AM" src="https://user-images.githubusercontent.com/114938895/213734761-d2a75873-afdb-471a-a882-158095fdc8cf.png">

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] - Windows can behave unexpectedly.

Thanks, for the implementation the library is so nice, if you can at least point me out which can be the issue is fine, don't need a large answer, only need to know more about the internal of virtuoso.

**Comment by EvgenyArtemov** [comment]
It works with useQuery perfectly, if you want to implement endless scrolling please use `fetchMore` method from Apollo client and follow to the docs.
